cluster/mesos/docker/util.sh

```diff
@@ -150,6 +149,37 @@ function cluster::mesos::docker::run_in_docker_cagen {
   return "${exit_status}"
 }
 
+# Run kube-keygen.sh inside docker.
+function cluster::mesos::docker::run_in_docker_keygen {
```

Comment:
Take a look at https://github.com/docker/docker/issues/3793#issuecomment-143592647 and https://github.com/krallin/tini to get working signals in docker-run.